LICENSING_POLICY.md

```diff
@@ -0,0 +1,10 @@
+Thank you for contributing to DataCommons.org. Before you add a new third-party dataset, please be sure that the data belongs to one of the below categories or is licensed under one of the approved licenses.
```

Comment:
Consider adding a title to this page

Should we link datacommons.org?

Reply:
`# Permitted Datasets` ?

I can add a link.

Reply:
Maybe `# Data Commons License Policy` or `# Data Commons Licensing Policy` is safer (based on the name of the doc I was given)